addons/sms/models/sms_sms.py

```diff
@@ -135,6 +135,7 @@ def _postprocess_iap_sent_sms(self, iap_results, failure_reason=None, delete_all
                         'failure_type': self.IAP_TO_SMS_STATE[state] if state != 'success' else False,
                         'failure_reason': failure_reason if failure_reason else False,
                     })
+        self.mail_message_id._notify_message_notification_update()
```

Comment:
Why this change ?

Reply:
The web client was out of date in that case because the notification wasn't sent after an update of the notification record just above.